rs/registry/canister-client/src/stable_canister_client/tests.rs

```diff
@@ -429,3 +431,370 @@ fn test_caching_behavior_of_get_latest_version() {
     let current_latest = client.get_latest_version();
     assert_eq!(current_latest, RegistryVersion::new(4));
 }
+
+fn registry_value(version: u64, value: &[u8]) -> RegistryValue {
+    RegistryValue {
+        value: value.to_vec(),
+        version,
+        deletion_marker: value.is_empty(),
+    }
+}
+
+fn registry_delta(key: &str, values: &[RegistryValue]) -> RegistryDelta {
```

Comment:
This is great. Converting existing tests to use the registry_delta function and registry_value function would maybe be beneficial?

Reply:
Agreed, I migrated the rest of the tests to use the same helpers